<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowWeekPlaner</class>
 <widget class="QMainWindow" name="MainWindowWeekPlaner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>764</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="copyto">
         <property name="text">
          <string>copyto</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="new_2">
         <property name="text">
          <string>new</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="delete_2">
         <property name="text">
          <string>delete</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clear">
         <property name="maximumSize">
          <size>
           <width>591</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>clear</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="WeekPlaner" name="weekplaner" native="true">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>400</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WeekPlaner</class>
   <extends>QWidget</extends>
   <header location="global">weekplaner.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
